org.alienarena.alienarena.yaml

```diff
@@ -0,0 +1,35 @@
+id: org.alienarena.alienarena
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: /app/src/alienarena/aa/source/bin/alienarena
+modules:
+  - name: alienarena
+    buildsystem: simple
+    no-autogen: true
+    sources:
+      - type: svn
+        url: svn://svn.icculus.org/alienarena/tags/7.71.4
```

Comment:
this is missing the svn revisions.

Reply:
This tag already points to a specific revision in svn.
How would this work if I recreate the tag based on a newer revision? I sometimes do this to include extra changes in the 7.71.4 release (mostly not major changes).

Reply:
that's not how svn work.

> How would this work if I recreate the tag based on a newer revision?

In the case of svn by specifying the version it would still build the same code, which is what we want.
Retagging IS a bad practice. You would need to update the revision to take up the new changes.

Reply:
it doesn't matter whether you are or not. You need to specify a revision.